src/com/tony/server/response/LogResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.tony.server.response;
+
+import com.tony.server.Logger;
+
+import java.util.HashMap;
+
+public class LogResponse extends Response{
+    private final Logger logger;
+
+    public byte[] respond(){
+        setBody(logger.getLog());
+        return ResponseBuilder.buildResponse(getStatusLine(),
+                getHeaders(), getBody()).getBytes();
+    } 
+    
+    public LogResponse(Logger logger) {
+        this.logger = logger;
+        setStatusLine("HTTP/1.1 200 OK");
```

Comment:
As far as your headers go, it seems like its pretty common for you to not set any headers, so maybe pull your default into your Response class.